### PR TITLE
Basic ability to send lipsync data to websocket when using Azure

### PR DESCRIPTION
**Pull request by ripperdoc**
- Collect lipsync events in Azure synthesizer
- Expose the events by updating SynthesisResult
- Send to websocket by modifying AudioMessage

### CODE REVIEW

File: vocode/streaming/synthesizer/azure_synthesizer.py [hunk 1]
```diff
@@ -278,9 +283,12 @@ async def chunk_generator(
                 yield SynthesisResult.ChunkResult(chunk_transform(audio_buffer), False)
 
         word_boundary_event_pool = WordBoundaryEventPool()
+        viseme_events = []
         self.synthesizer.synthesis_word_boundary.connect(
             lambda event: self.word_boundary_cb(event, word_boundary_event_pool)
         )
+        # print("Viseme event received: audio offset: {}ms, viseme id: {}.".format(evt.audio_offset / 10000, evt.viseme_id))
```
Comment (manuel-fictive) [inline vocode/streaming/synthesizer/azure_synthesizer.py:290]: Should we keep this print?

File: vocode/streaming/synthesizer/base_synthesizer.py [hunk 2]
```diff
@@ -63,9 +63,11 @@ def __init__(
         self,
         chunk_generator: AsyncGenerator[ChunkResult, None],
         get_message_up_to: Callable[[int], str],
+        get_lipsync_events: Callable[[int, int], []] = None,
```
Comment (manuel-fictive) [inline vocode/streaming/synthesizer/base_synthesizer.py:66]: Pylance is throwing here a warning on `[]` inside `[[int, int], []]` saying: List expression not allowed for this argument of type Pylance[reportInvalidTypeForm](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportInvalidTypeForm)

File: vocode/streaming/synthesizer/azure_synthesizer.py (hunk 1 above)
Comment (ripperdoc) [reply vocode/streaming/synthesizer/azure_synthesizer.py:290]: I'll remove it

File: vocode/streaming/synthesizer/base_synthesizer.py (hunk 2 above)
Comment (ripperdoc) [reply vocode/streaming/synthesizer/base_synthesizer.py:66]: I don't get this issue, so we must have differently setup Pylance. I don't see what the issue would be so will skip for now.

**Pull request merged**